specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/2016-08-10/service.json

```diff
@@ -18,6 +18,46 @@
     "application/json"
   ],
   "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{resourceName}/replicationVaultHealth": {
```

Comment:
Is this API also giving the health summary for all the child resources of vault (fabric/ protectedItem etc)? If yes, then does the caller of this API need to have read access on all the underlying resources as well.  Also, from the API signature it is unclear. Moreover, in the payload it will be unclear which "replicationItem" does a particular issue correspond to? Am I missing something?